src/main/java/eightbit/EightBitException.java

```diff
@@ -0,0 +1,25 @@
+package eightbit;
+
+/**
+ * Exception that is used to handle incorrect inputs from user.
+ */
+public class EightBitException extends Exception {
+    private String errorMessage;
+
+    /**
+     * Constructs the exception with the error message.
+     *
+     * @param errorMessage Error message to be shown to user.
+     */
+    public EightBitException(String errorMessage) {
+        this.errorMessage = errorMessage;
```

Comment:
I think you should use the constructor of the parent class.
```suggestion
        super(errorMessage);
```